### PR TITLE
FAT-977 mod-kb-ebsco-java: Implement API Karate tests - Resources

### PR DESCRIPTION
**Pull request by Shans-Kaluhin**
## Purpose
Karate implementation based on test plans is created

## API Documentation: 
  [**Resourcres**](https://s3.amazonaws.com/foliodocs/api/mod-kb-ebsco-java/resources.html#)

## Learning
  [FAT-977](https://issues.folio.org/browse/FAT-977)

### CODE REVIEW

File: mod-kb-ebsco-java/src/main/resources/domain/mod-kb-ebsco-java/features/setup/destroy.feature [hunk 1]
```diff
@@ -7,15 +7,22 @@ Feature: Destroy test data for kb-ebsco-java
```
Comment (oleksandr-dekin) [inline mod-kb-ebsco-java/src/main/resources/domain/mod-kb-ebsco-java/features/setup/destroy.feature:7]: can we add @DestroyResource here:
```
   #destroy recourse
    Given path '/eholdings/resources', resourcesId
    When method DELETE
    Then status 204
    * eval sleep(10000)
```

File: mod-kb-ebsco-java/src/main/resources/domain/mod-kb-ebsco-java/features/resources.feature [hunk 2]
```diff
@@ -4,67 +4,250 @@ Feature: Resources
...
-  Scenario: POST Resources should return 422 if Package id is invalid
-    * print 'undefined'
-
-  @Undefined
+  @Positive
+  Scenario: POST Resources with 200 on success
+    Given path '/eholdings/resources'
+    And request read(samplesPath + 'resources.json')
+    When method POST
+    Then status 200
```
Comment (natali-zaitseva) [inline mod-kb-ebsco-java/src/main/resources/domain/mod-kb-ebsco-java/features/resources.feature:19]: shouldn't it be 201 on post?

Comment (Shans-Kaluhin) [reply mod-kb-ebsco-java/src/main/resources/domain/mod-kb-ebsco-java/features/resources.feature:19]: It should.
But mod-kb-ebsco returns 200 on post requests: [Api Documentation](https://s3.amazonaws.com/foliodocs/api/mod-kb-ebsco-java/resources.html#eholdings_resources_post)

File: mod-kb-ebsco-java/src/main/resources/domain/mod-kb-ebsco-java/features/resources.feature [hunk 3]
```diff
@@ -4,67 +4,250 @@ Feature: Resources
...
+
+    #waiting for resources creation
+    * eval sleep(10000)
+    * def resourcesId = response.data.id
+
+    #destroy recourse
```
[the remark this answers is not in the document]
Comment (Shans-Kaluhin) [reply mod-kb-ebsco-java/src/main/resources/domain/mod-kb-ebsco-java/features/resources.feature:27]: Now it removing in the script after each scenario, and gets resourceId from karate properties

**Review by psmagin — APPROVED** [review]

**Review by natali-zaitseva — APPROVED** [review]

**Pull request merged**